files/jquery.serialscroll/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",

```

Comment:
if you are going to use bower then the `name` parameter is used to query bower for the package. `jquery.serialScroll` is not the correct name in that case. You can switch to github instead if you want